BambooTracker/res/lang/bamboo_tracker_fr.ts

```diff
@@ -551,32 +551,32 @@
     <message>
         <location filename="../../gui/fm_envelope_set_edit_dialog.ui" line="14"/>
         <source>Edit FM envelope set</source>
-        <translation type="unfinished"></translation>
+        <translation>Éditer le format d&apos;enveloppe FM</translation>
```

Comment:
format -> ensemble ?

Reply:
It's really a "format" definition we're editing in this context and not something else.
If a nearer definition for "set" could be given, it would be something along the line of "jeu de valeurs".

(actually more precisely: "les types du jeu de valeur", since we're talking meta in this context. it's a bit an akward thing to formulate)

Reply:
ok